src/app/body/board/performance-column-chart/performance-column-chart.component.ts

```diff
@@ -0,0 +1,120 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { AngularFirestore } from '@angular/fire/firestore';
+import { Tasks } from 'src/app/Interface/TasksInterface';
+import { ApplicationSettingsService } from 'src/app/services/applicationSettings/application-settings.service';
+import { AuthService } from 'src/app/services/auth.service';
```

Comment:
There is no use of ApplicationSettingsService and AuthService. Please remove these imports